examples/playbooks/hosts.ini

```diff
@@ -27,7 +27,7 @@ ansible_user=root
 ansible_ssh_private_key_file='../ssh-key'
 ansible_python_interpreter=python3
 kubernetes_kubeconfig_path='./'
-kubernetes_version='1.20.15'
+kubernetes_version='1.23.12'
```

Comment:
Why not 1.24.something?

Reply:
In the example I also show the upgrade procedure